english/dream/swim/swim.md

```diff
@@ -11,3 +11,15 @@ A pain shoots up your legs. You glance down to where your feet should be only to
 flipper wrapped tightly in the ends of licorice whips.
 
 You are trapped. Your dream is going horribly wrong. You have been ensnared by... 
+
+... Ursula the sea witch!
+
+She is enormous, with a mouth large enough to eat you in three bites! Her licorice-whip hair is 
+wrapped around your flipper, and she begins to pull you towards her, reeling in the licorice like 
+fishing line.
+
+What to do!?
+
+[Attempt to chew through the licorice lines ensnaring you.](/english/dream/swim/swim-trap/chew.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.